developer-docs-site/docs/guides/sponsored-transactions.md

```diff
@@ -0,0 +1,70 @@
+* [MerkleTrade](https://merkle.trade/) offers low cost trading to those with Ethereum wallets by creating an Aptos wallet for users and covering all transaction fees so that the user does not need to acquire utility tokens for Aptos.
+* Community engagement applications like [Graffio](https://medium.com/aptoslabs/graffio-web3s-overnight-sensation-81a6cf18b626) offered to cover transaction fees for custodial accounts to support the collaborative drawing application for those without wallets.
+
+## Process Overview
+
+The process for sending a sponsored transaction follows:
+* The sender(s) of the transaction agree upon an operation, as defined by a `RawTransaction`.
+* The sender(s) generate a `RawTransactionWithData::MultiAgentWithFeePayer` structure
+  * As noted below, prior to the framework 1.8 release, this must contain the fee payer's address.
+  * After framework release 1.8, this can optionally be set to `0x0`.
+* The sender(s) can forward the signed transaction to the fee payer who signs with their signature and forwards it to the blockchain.
+* Upon execution of the transaction, the sequence number of the primary sending account is incremented and all gas fees are deducted and all refunds are sent to the gas fee payer.
+
+Alternatively, if the fee payer knows the operation and all parties involved, they could sign a transaction and forward it back to the parties to sign.
+
+## Technical Details
+
+In Aptos, a sponsored transaction reuses the same SignedTransaction as any other user transaction:
+```rust
+pub struct SignedTransaction {
+    /// The raw transaction
+    raw_txn: RawTransaction,
+
+    /// Public key and signature to authenticate
+    authenticator: TransactionAuthenticator,
+}
+```
+
+The difference is in the `TransactionAuthenticator`, which stores the authorization from the fee payer of the transaction to extract utility fees from their account:
+```rust
+pub enum TransactionAuthenticator {
+...
+    /// Optional Multi-agent transaction with a fee payer.
+    FeePayer {
+        sender: AccountAuthenticator,
+        secondary_signer_addresses: Vec<AccountAddress>,
+        secondary_signers: Vec<AccountAuthenticator>,
+        fee_payer_address: AccountAddress,
+        fee_payer_signer: AccountAuthenticator,
+    },
+...
+}
+```
+
+To prepare a sponsored transaction for an account, the account must first exist on-chain. This is a requirement that is being removed with the 1.8 framework release.
+
+As of the 1.8 framework release, an account need not exist on-chain. However, the first transaction for an account requires enough gas to not only perform the operation dictated by the transaction but also cover the costs associated with account creation, even if an account already exists. This requirement is expected to subside with advances in our account model.
```

Comment:
It's already out for voting, maybe just want to shortcut straight to post 1.8 release?  (if it does get approved)